views/index.hbs

```diff
@@ -8,6 +8,17 @@
 </head>
 
 <body>
+  {{#each gameState.board}}
+  {{#each this}}
+  <form method="get" action="/">

```

Comment:
I don't understand why you "draw" the board twice, once with a hidden form and once with divs. Why not draw it once with a visible form instead?

Reply:
This is a solution I tried to make something work quickly. My first attempts in the way you suggest broke the style of the board. But i can retry now to merge cells+forms.